design/aws-ecs-priv-registry-support.md

```diff
@@ -75,56 +69,39 @@ export interface IContainerImage {
 ```
 
 
-## Part 2: Extend `DockerHubImage` class to include optional creds
+# Part 2: Rename `DockerHubImage` class to `WebHostedImage`, add optional creds
 
-The `DockerHubImage` construct, which currently takes a string for image name, can be extended to take in an (optional) "repositoryCredentials" field of type `IRepositoryCreds`:
+The `DockerHubImage` construct will be renamed to `WebHostedImage`, and augmented to take in optional "credentials" via keyword props:
```

Comment:
I don't mind this rename at all, but being the pedant that I am, is "Web" the right term to use? It doesn't *really* have anything to do with a linked network of HTML pages...  `InternetHostedImage` would be a more correct term.

Not saying better, just more correct.